requirements.txt

```diff
@@ -24,3 +24,4 @@ raven==5.23.0
 demjson==2.2.4
 greenlet==0.4.9
 yoyo-migrations==5.0.3
+markdown_toclify==0.1.7
```

Comment:
A table of content allows for an effective way to find the needed information. I ll admit this one is at the early stage...